priv/templates/homepage-feed/user-feed.tpl

```diff
@@ -5,17 +5,17 @@
 
         <div>
             <h2 class="bordered-title">Actuele updates en nieuws uit jouw kennisgroepen en regio's</h2>
+            {% if m.search[{query cat=['event', 'contribution', 'remark'] sort='-rsc.created' is_published="true" content_group=m.acl.user.o.hascollabmember region=m.acl.user.s.hasregion pagelen=5 }] as results %}
```

Comment:
`region` by itself is not a search argument (I suspect that if you look at your console it'll also have some message about ignoring it), what we could use instead is the [`hasobject`/`hassubject` standard filters](https://zotonic.com/docs/1638/search).

However, I'm not sure that will get the desired effect, as this filter would be in an "and" with the rest. I don't see the exact request in the issue, could you please point that out?

Reply:
Ah yeah, i'll adjust. Basically they want to see either kennisgroep items or region items https://driebit.atlassian.net/browse/KC-640

Reply:
Ah, I see, that makes it slightly more complicated, because as I was saying filters usually combine in an "and", meaning that using `hasobject`/`hassubject` with `content_group` would select only the resources that are both in one of the regions and in one of the content groups at the same time.

I suspect this would be easier to handle by adding an option in the backend, do you mind if I give it a shot and push back a commit to this branch?

Reply:
@EdwardAbboud done in cf379cd714759b78a7179cbe4c31886ee56be80c, please check if it works as expected 🙂